internal/app/grpcserver/server.go

```diff
@@ -96,11 +117,28 @@ func Run(ctx context.Context, args []string, inReader io.ReadCloser, outWriter i
 	}
 }
 
-func createServer() (*grpc.Server, error) { //nolint:unparam
-	server := grpc.NewServer(
-		grpc.UnaryInterceptor(serverNameUnaryInterceptor),
-		grpc.StreamInterceptor(serverNameStreamInterceptor),
-	)
-	v1.RegisterConformanceServiceServer(server, NewConformanceServiceServer())
-	return server, nil
+func runGRPCWebServer(ctx context.Context, server *grpc.Server, listener net.Listener) error {
+	httpServer := http.Server{
+		Handler:           h2c.NewHandler(grpcweb.WrapServer(server), &http2.Server{}),
```

Comment:
This `grpcweb.WrapServer` call is the special-sauce that makes this work with grpc-web.